agent/xds/endpoints.go

```diff
@@ -443,17 +454,18 @@ func makeLoadAssignment(clusterName string, endpointGroups []loadAssignmentEndpo
 
 		for _, ep := range endpoints {
 			// TODO (mesh-gateway) - should we respect the translate_wan_addrs configuration here or just always use the wan for cross-dc?
-			addr, port := ep.BestAddress(localDatacenter != ep.Node.Datacenter)
-			healthStatus, weight := calculateEndpointHealthAndWeight(ep, endpointGroup.OnlyPassing)
+			// TODO markan revisit this, because I'm not sure we have to handle sockets here.
```

Comment:
Practically speaking, no. But I suppose you could leave the possibility in so this enclosing `makeLoadAssignment` function is address-agnostic?

Reply:
I decided to remove this code in the interests of simplicity.